### PR TITLE
Add supported values of APISERVER, CONTROLLER_MANAGER, and SCHEDULER to logging_config 

### PR DESCRIPTION
**Pull request by yungdarek122**
Add supported values of APISERVER, CONTROLLER_MANAGER, and SCHEDULER to logging_config. These values are supported after the GA of the feature of exporting GKE control plane log to customers.

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [N/A - not adding any new field] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [N/A] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

```release-note:enhancement
container: added support for additional values `APISERVER`, `CONTROLLER_MANAGER`, and `SCHEDULER` in `google_container_cluster.monitoring_config`
```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 1]
```diff
@@ -523,8 +523,8 @@ as "Intel Haswell" or "Intel Sandy Bridge".
 
 <a name="nested_logging_config"></a>The `logging_config` block supports:
 
-*  `enable_components` - (Required) The GKE components exposing logs. Supported values include:
-`SYSTEM_COMPONENTS` and `WORKLOADS`.
+*  `enable_components` - (Optional) The GKE components exposing logs. Supported values include:
```
Comment (megan07) [inline mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown:526]: This seems to still be [required](https://github.com/GoogleCloudPlatform/magic-modules/pull/6772/files?diff=unified&w=1#diff-c8cc57371d31377352c614135ade5ab3273f80b595ee86b222b1271ab6d5b984L717).

**Review by megan07 — CHANGES_REQUESTED** [review]
One quick change, otherwise looks good, thanks!

File: mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown (hunk 1 above)
Comment (yungdarek122) [reply mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown]: Thanks, changed it to "required"

**Review by megan07 — APPROVED** [review]
Thanks!

**Pull request merged**